samples/csharp_dotnetcore/53.teams-messaging-extensions-action-preview/Bots/TeamsMessagingExtensionsActionPreviewBot.cs

```diff
@@ -143,17 +143,32 @@ protected override async Task<MessagingExtensionActionResponse> OnTeamsMessaging
             var activityPreview = action.BotActivityPreview[0];
             var attachmentContent = activityPreview.Attachments[0].Content;
             var previewedCard = JsonConvert.DeserializeObject<AdaptiveCard>(attachmentContent.ToString(), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore });
-
+            previewedCard.Version = "1.0";
```

Comment:
This seems a bit out of place - do you have to use a v1.0 adaptive card for this? Why? If the version needs to be set, I think it should be set in the adaptiveCardHelper where the card is created, right?

Reply:
Thanks @clearab for your comments, I will check and update it accordingly